roles/tas_single_node/tasks/certificates.yml

```diff
@@ -1189,3 +1205,4 @@
       changed_when: false
       failed_when: tsa_key_passphrase.stdout == "failed"
       when: tas_single_node_tsa.signer_private_key is defined and tas_single_node_tsa.signer_private_key != ""
+      no_log: true
```

Comment:
Would the two above tasks actually print out something sensitive? I think they only reference the files with sensitive content, but they won't actually print it, will they?